content/en/docs/reference/glossary/trust-domain.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Trust Domain
+---
+
+An [identity](/docs/reference/glossary/#identity) contains a substring
```

Comment:
We should probably lead with the definition of trust domain.  Maybe something like:

```
A trust domain is a globally unique name used to create identities. Every Istio mesh has exclusive ownership of a single trust domain, which is used to create all identities within the mesh. 
```

I think that should be sufficient.  WDYT?

Reply:
@nmittler Yes, I like your wording but would still add a link:
```
A trust domain is a globally unique name used to create
[identities](/docs/reference/glossary/#identity). Every Istio
mesh has exclusive ownership of a single trust domain, which is used to create
all identities within the mesh.
```

Reply:
Maybe a simplified version:

```
A trust domain is a globally unique name that Istio uses to create all identities within a mesh. Every mesh has an exclusive trust domain. 
```
WDYT

Reply:
And yes definitely with a link.

Reply:
Hmm ... yours is more brief, for sure. But I feel it loses the explicit statement that all identities in the mesh are created with the same trust domain. Yes it's implied, but you have to parse it in your head a little to get there.

Reply:
Actually, looking at it again, I think your definition is fine. 

@frankbu WDYT?

Reply:
Yep, the simpler version looks fine to me as long as it has the link from "identities".